<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>502</height>
   </rect>
  </property>
  <property name="baseSize" >
   <size>
    <width>780</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Mudlet 1.0 - alpha 1</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/mudlet_main_16px.png</normaloff>:/mudlet_main_16px.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>14</y>
     <width>750</width>
     <height>488</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font" >
       <font>
        <pointsize>6</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy" >
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="viewMode" >
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font" >
    <font>
     <family>DejaVu Sans</family>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="allowedAreas" >
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2" >
   <property name="geometry" >
    <rect>
     <x>4</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3" >
   <property name="geometry" >
    <rect>
     <x>18</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_4" >
   <property name="geometry" >
    <rect>
     <x>32</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_5" >
   <property name="geometry" >
    <rect>
     <x>46</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_5</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_6" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_6</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_7" >
   <property name="geometry" >
    <rect>
     <x>74</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_7</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_8" >
   <property name="geometry" >
    <rect>
     <x>88</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_8</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_9" >
   <property name="geometry" >
    <rect>
     <x>102</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_9</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_10" >
   <property name="geometry" >
    <rect>
     <x>116</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_10</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_11" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_11</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_12" >
   <property name="geometry" >
    <rect>
     <x>144</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_12</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_13" >
   <property name="geometry" >
    <rect>
     <x>158</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_13</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_14" >
   <property name="geometry" >
    <rect>
     <x>172</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_14</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_15" >
   <property name="geometry" >
    <rect>
     <x>186</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_15</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_16" >
   <property name="geometry" >
    <rect>
     <x>200</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_16</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_17" >
   <property name="geometry" >
    <rect>
     <x>214</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_17</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_18" >
   <property name="geometry" >
    <rect>
     <x>228</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_18</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_19" >
   <property name="geometry" >
    <rect>
     <x>242</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_19</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_20" >
   <property name="geometry" >
    <rect>
     <x>256</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_20</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_21" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_21</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_22" >
   <property name="geometry" >
    <rect>
     <x>284</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_22</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_23" >
   <property name="geometry" >
    <rect>
     <x>298</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_23</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_24" >
   <property name="geometry" >
    <rect>
     <x>312</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_24</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_25" >
   <property name="geometry" >
    <rect>
     <x>326</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_25</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_26" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_26</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_27" >
   <property name="geometry" >
    <rect>
     <x>354</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_27</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_28" >
   <property name="geometry" >
    <rect>
     <x>368</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_28</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_29" >
   <property name="geometry" >
    <rect>
     <x>382</x>
     <y>0</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_29</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_30" >
   <property name="geometry" >
    <rect>
     <x>396</x>
     <y>0</y>
     <width>354</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_30</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/preferences-web-browser-cache.png</normaloff>:/icons/preferences-web-browser-cache.png</iconset>
   </property>
   <property name="text" >
    <string>Connect</string>
   </property>
  </action>
  <action name="actionTriggers" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/tools-wizard.png</normaloff>:/icons/tools-wizard.png</iconset>
   </property>
   <property name="text" >
    <string>Triggers</string>
   </property>
  </action>
  <action name="actionTimers" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/chronometer.png</normaloff>:/icons/chronometer.png</iconset>
   </property>
   <property name="text" >
    <string>Timers</string>
   </property>
  </action>
  <action name="actionScripts" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/document-properties.png</normaloff>:/icons/document-properties.png</iconset>
   </property>
   <property name="text" >
    <string>Scripts</string>
   </property>
  </action>
  <action name="actionAliases" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/system-users.png</normaloff>:/icons/system-users.png</iconset>
   </property>
   <property name="text" >
    <string>Aliases</string>
   </property>
  </action>
  <action name="actionButtons" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/rating.png</normaloff>:/icons/rating.png</iconset>
   </property>
   <property name="text" >
    <string>Buttons</string>
   </property>
  </action>
  <action name="actionKeys" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/preferences-desktop-keyboard.png</normaloff>:/icons/preferences-desktop-keyboard.png</iconset>
   </property>
   <property name="text" >
    <string>Keys</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/help-hint.png</normaloff>:/icons/help-hint.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionNotes" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/view-pim-notes.png</normaloff>:/icons/view-pim-notes.png</iconset>
   </property>
   <property name="text" >
    <string>Notes</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="../mudlet_alpha.qrc" >
     <normaloff>:/icons/mudlet_main_48px.png</normaloff>:/icons/mudlet_main_48px.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../mudlet_alpha.qrc" />
 </resources>
 <connections/>
</ui>
